### PR TITLE
[project-base] [SSP-2589] cypress improvements

### PR DESCRIPTION
**Pull request by sebaholesz**
#### Description, the reason for the PR
This PR is the next phase of continual cypress improvements

#### Check the appropriate checkboxes below, please

- [ ] [BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/) 
- [ ] New feature 
- [x] I have read and signed [License Agreement for contributions](https://www.shopsys.com/license-agreement)

----
:globe_with_meridians: Live Preview:
  - https://sh-ssp-2589-cypress-improvements.odin.shopsys.cloud
  - https://cz.sh-ssp-2589-cypress-improvements.odin.shopsys.cloud

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,6 +48,24 @@ run-acceptance-tests-base:
 run-acceptance-tests-actual:
 	$(call run_acceptance_tests,actual)
 
+define selected_acceptance_tests
```
Comment (JanMolcik) [inline Makefile:51]: Why do we maintain two almost identical Makefiles?

File: project-base/storefront/cypress/e2e/visits/simpleVisitsWithScreenshots.cy.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { url } from 'fixtures/demodata';
```
Comment (JanMolcik) [inline project-base/storefront/cypress/e2e/visits/simpleVisitsWithScreenshots.cy.ts:1]: Why do we have two almost identical files `project-base/storefront/cypress/e2e/visits/simpleVisitsWithScreenshots.cy.ts` and `project-base/storefront/cypress/e2e/visits/simpleVisitsWithScreenshots.ts`?

**Review by JanMolcik — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (sebaholesz) [reply Makefile:51]: One is for us, the other one is for projects (the one in project-base).

File: project-base/storefront/cypress/e2e/visits/simpleVisitsWithScreenshots.cy.ts (hunk 2 above)
Comment (sebaholesz) [reply project-base/storefront/cypress/e2e/visits/simpleVisitsWithScreenshots.cy.ts:1]: This is a mistake, thanks for catching this. I have removed one of them.

**Review by JanMolcik — APPROVED** [review]

**Pull request merged**